common/src/main/scala/org/genivi/sota/http/AuthNamespaceDirectives.scala

```diff
@@ -46,9 +44,9 @@ object AuthedNamespaceScope {
       case x if x.startsWith(namespacePrefix) => x.substring(namespacePrefix.length)
     }
     if (nsSet.size == 1) {
-      AuthedNamespaceScope(Namespace(nsSet.toVector(0)), token.scope, owned = false)
+      AuthedNamespaceScope(Namespace(nsSet.toVector(0)), Some(token.scope))
     } else {
-      AuthedNamespaceScope(Namespace(token.subject.underlying), token.scope, owned = true)
+      AuthedNamespaceScope(Namespace(token.subject.underlying), Some(token.scope))
```

Comment:
This is new semantic, do you really want to have ```Some(token.scope)``` or do you want ```None``` which is what corresponds to ```owned = true```?